File: aat-lib/src/main/java/ch/bailu/aat_lib/service/elevation/updater/ElevationUpdaterClient.java
package ch.bailu.aat_lib.service.elevation.updater;

import ch.bailu.aat_lib.app.AppContext;
import ch.bailu.aat_lib.service.elevation.tile.Dem3Tile;

public interface ElevationUpdaterClient {
    void updateFromSrtmTile(AppContext appContext, Dem3Tile tile);
}

File: aat-lib/src/main/java/ch/bailu/aat_lib/service/elevation/updater/ElevationUpdaterClient.kt
package ch.bailu.aat_lib.service.elevation.updater

import ch.bailu.aat_lib.app.AppContext
import ch.bailu.aat_lib.service.elevation.tile.Dem3Tile

interface ElevationUpdaterClient {
    fun updateFromSrtmTile(appContext: AppContext, tile: Dem3Tile)
}
